### PR TITLE
Support prefixes with args

### LINKED ISSUE #41: prefix doesn't support args
First off, thanks for adding the `prefix:` option. It definitely simplified things for projects with graphical dependencies. Unfortunately it doesn't allow for prefixes with arguments. For example, I tried using [nix](https://nixos.org/) to install non-Julia dependencies by specifying:
```
      - uses: julia-actions/julia-runtest@v1
        with:
          prefix: nix-shell --run
```
In the shell this would done like "nix-shell --run 'julia -e 'using Pkg; Pkg.test()'". Unfortunately, `julia-runtest` treats `nix-shell --run` as the command, resulting in the following error:
```
/home/runner/work/_temp/dd15467b-7a13-4a49-89b9-3405b61d48af.sh: line 9: nix-shell --run: command not found
```
I think this would be fixed by changing:
```
        [[ -n $prefix ]] && julia_cmd=( "$prefix" "${julia_cmd[@]}" )
```
to
```
        [[ -n $prefix ]] && julia_cmd=( ${prefix[@]} "${julia_cmd[@]}" )
```

### PR DESCRIPTION
**Pull request by IanButterworth**
Implements @colinxs's suggestion in https://github.com/julia-actions/julia-runtest/issues/41

Fixes #41

### CODE REVIEW

**Comment by IanButterworth** [comment]
Doesn't seem quite right yet https://github.com/IanButterworth/Example.jl/pull/1

**Comment by IanButterworth** [comment]
ok, this is working over in IanButterworth/Example.jl#1 with both single prefixes and prefixes with arguments

Is use of `eval` like this ok?

cc. @giordano

**Comment by Octogonapus** [comment]
Bumping this so that https://github.com/julia-actions/julia-buildpkg/pull/16 can eventually get merged.

**Comment by IanButterworth** [comment]
@SaschaMann 🙏🏻

File: action.yml [hunk 1]
```diff
@@ -51,12 +51,16 @@ runs:
         JULIA_PKG_SERVER: ""
     - run: |
         # The Julia command that will be executed
-        julia_cmd=( julia --color=yes --depwarn=${{ inputs.depwarn }} --inline=${{ inputs.inline }} --project=${{ inputs.project }} -e 'import Pkg;include(joinpath(ENV["GITHUB_ACTION_PATH"], "kwargs.jl"));kwargs = Kwargs.kwargs(;coverage = :(${{ inputs.coverage }}),force_latest_compatible_version = :(${{ inputs.force_latest_compatible_version }}), julia_args = ["--check-bounds=${{ inputs.check_bounds }}"]);Pkg.test(; kwargs...)' )
+        julia_cmd="julia --color=yes --depwarn=${{ inputs.depwarn }} --inline=${{ inputs.inline }} --project=${{ inputs.project }} -e 'import Pkg;include(joinpath(ENV[\"GITHUB_ACTION_PATH\"], \"kwargs.jl\"));kwargs = Kwargs.kwargs(;coverage = :(${{ inputs.coverage }}),force_latest_compatible_version = :(${{ inputs.force_latest_compatible_version }}), julia_args = [\"--check-bounds=${{ inputs.check_bounds }}\"]);Pkg.test(; kwargs...)'"
```
Comment (SaschaMann) [inline action.yml:54]: Sorry I forgot about this for so long.

My understanding of bash is that for commands like this, parantheses should be used. The if/then block below deals with the main issue we ran into before when using double quotes but there might be other unintentional consequences I'm not aware of. So I'd prefer to stick to the best practice that was recommended back then rather switching to double quotes.

The second suggestion in #41 seems like it should work without going back to double quotes.

**Review by SaschaMann — CHANGES_REQUESTED** [review]

**Comment by lassepe** [comment]
May I carefully bump this PR? I could really use this feature in a range of package CIs. Seems like it's only a small change away from being mergeable.

**Pull request closed without merging**